core/error/error_macros.h

```diff
@@ -403,6 +403,19 @@ void _err_flush_stdout();
 	} else                                                                                             \
 		((void)0)
 
+/**
+ * Ensures `m_cond` is false.
+ * If `m_cond` is true, the current function returns, and the error is printed directly
+ * to stderr, bypassing the MessageQueue. This is useful to use in core templates
+ * used by MessageQueue itself, to prevent infinite error loops.
+ */
+#define ERR_FAIL_COND_STDOUT(m_cond)                                                                          \
+	if (unlikely(m_cond)) {                                                                            \
+		fprintf(stderr, "ERROR: Condition \"%s\" is true.\n   at: %s (%s:%i)\n", _STR(m_cond), FUNCTION_STR, __FILE__, __LINE__); \
+		return;                                                                              \
+	} else                                                                                   \
+ 		((void)0)
```

Comment:
```suggestion
```suggestion
#define ERR_FAIL_COND_STDOUT(m_cond)                                                                                              \
	if (unlikely(m_cond)) {                                                                                                       \
		fprintf(stderr, "ERROR: Condition \"%s\" is true.\n   at: %s (%s:%i)\n", _STR(m_cond), FUNCTION_STR, __FILE__, __LINE__); \
		return;                                                                                                                   \
	} else                                                                                                                        \
		((void)0)
```
Formatting to fix CI